bvh-f64/Cargo.toml

```diff
@@ -0,0 +1,59 @@
+[package]
+name = "bvh-f64"
```

Comment:
Do we really need a specialized crate for this? Can't we use traits for this to have Rust do the specialization?

Reply:
So I tried a few approaches to this. It might be a possibility with nalgebra backing things to have the float be a generic on everything, but with glam there's no trait to abstract dvec3 from vec3 so we would have to wrap that ourselves. This trick using separate crates in the workspace is something I got from the author of nalgebra and how he builds rapier (2d 3d 32 and 64bit). When the workspace is all setup with this cargo test runs the test for all the versions and you'll get compiler errors from rust analyzer for both at the same time

Reply:
I would very much like to see this move to nalgebra, it would make the usage much cleaner via `Bvh<T>`. This would definitely be possible with the nalgebra vector types and number bounds.

Heck, because of the AABB's, assuming there are no dot products/cross products (which I don't think there are - haven't checked), it should work with integer types as well, and maybe even unsigned types (but there would have to be better bounds checking).

I could try and help get this working with nalgebra as well.

Reply:
I think that would be pretty cool.

Reply:
I'm not sure how to approach this for now though, should I wait for these changes to get merged in?

Reply:
Hmmm it might be easier for me to port it now, and have this PR rebase on it. Otherwise we would be splitting code and then remerging.

Reply:
Has nalgebra fixed the issues that made this crate switch from nalgebra to glam previously?